Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -461,9 +491,29 @@ namespace gdjs {
       checkOrigin: boolean;
       onDone?: (status: 'logged' | 'errored' | 'dismissed') => void;
     }) {
-      const allowedOrigins = ['https://liluo.io', 'https://gd.games'];
-      // const allowedOrigins = ['localhost:4000'];
+      const onLoginSuccessFul = (event: MessageEvent) => {
+        login({
+          runtimeScene,
+          userId: event.data.body.userId,
+          username: event.data.body.username,
+          userToken: event.data.body.token,
+        });
+        focusOnGame(runtimeScene);
+        if (onDone) onDone('logged');
+      };
+
+      const onLoginWithoutLoginDialog = (event: MessageEvent) => {
```

Comment:
The name "WithoutLoginDialog" is a bit weird to me, let's try to find something else (i'm not sure what "LoginDialog" refers to)

Reply:
Yeah it's basically when the login happens without the authentication iframe/webview visible.
The consequence is that we don't need to close it, but more importantly, we don't close the banner and show a login success notification.
So it's when the login is done automatically without the user input. So maybe a `onLoginDoneAutomatically` with a comment specifying the cases (games-platform providing player token)